RELEASE_NOTES.md

```diff
@@ -6,18 +6,27 @@ Any new bugs, requests, or missing features should be reported as [Github issues
 
 ## Improvements:
 
-### Command line
- * Rename script to zcbor.
+### CDDL language
+ * Add support for 32-bit and 64-bit floating point numbers.
```

Comment:
nit: There's a hyphen here between "32/64" and "bit", but in other places I see around the various diffs (L30, L31, L83, for example) there is no hyphen used. Should we stick to particular way of writing these for consistency? If so then IMO the hyphen looks nice, but whichever you prefer.

Reply:
Changed all occurrences to use hyphen since that seems to be the norm when googling.